testing_data/bounding_box_dict.json

```diff
@@ -0,0 +1,21 @@
+{
+  "testing_data/sub-test001/anat/sub-test001_T1w.nii.gz":
+    [
+      7,
+      386,
+      113,
+      231,
+      1,
+      16
+    ],
+
+  "testing_data/sub-test001/anat/sub-test001_T2w.nii.gz":
+    [
+      7,
+      386,
+      113,
+      231,
+      1,
+      16
+    ]
+}
```

Comment:
Not relevant as it is in ivadomed/data-testing

Reply:
Should we delete it then?

Reply:
This one yes because I created it.

Reply:
Done